src/vm/virtualcallstub.cpp

```diff
@@ -2518,13 +2527,13 @@ VirtualCallStubManager::TraceResolver(
     CONSISTENCY_CHECK(CheckPointer(pMT));
 
 
-    DispatchSlot slot(pMT->FindDispatchSlot(token));
+    DispatchSlot slot(pMT->FindDispatchSlot(token, TRUE /* throwOnConflict */));
 
```

Comment:
throwing here is going to do something odd in the debugger when stepping into a failing interface dispatch. I'm not convinced throwing is wrong, but I'm not sure what it'll do.

Reply:
Good catch, added that to the debugger work tracking issue (#15866).